### ISSUE TITLE
can't see the map

### ISSUE THREAD

**Issue by lashot** [covidatlas/li#355]
Is it me? The map isn't loading anymore.

**Comment by jzohrab** [comment]
Yes it’s (hopefully temporarily) disabled. We launched a new data report
format and I had to hide the map.

**Comment by lashot** [comment]
Thank you.  Looking forward to seeing it again.

-L
